docs/topics/request-response.rst

```diff
@@ -1328,3 +1328,12 @@ XmlResponse objects
     line.  See :attr:`TextResponse.encoding`.
 
 .. _bug in lxml: https://bugs.launchpad.net/lxml/+bug/1665241
+
+JsonResponse objects
+--------------------
+
+.. class:: JsonResponse(url[, ...])
+
+    The :class:`JsonResponse` class is a subclass of :class:`TextResponse` which
+    adds encoding auto-discovering support by looking into the JSON declaration
+    line.  See :attr:`TextResponse.encoding`.
```

Comment:
I don't think this makes sense?

Reply:
Oh you're right, you think this might work:

```rst
    The :class:`JsonResponse` class is a subclass of :class:`TextResponse` which
    adds encoding auto-discovering support by looking for a MIME type in the
    `Content-Type` header that represent to a JSON object.  See :attr:`TextResponse.encoding`.
```